src/lib_ccaligner/params.h

```diff
@@ -9,11 +9,12 @@
 
 #include "commons.h"
 
-const std::string currentTime();
+std::string currentTime();
 
 class Params
 {
     std::string localTime;
+    void inputParams(int argc, char *argv[]);
```

Comment:
Though it definitely looks like a good idea, but unfortunately we can not do this. We need to be as flexible as possible. There are many scenarios in which the parameters need be changed mid-way and then brought into usage. Or may be process user input, modify it, and then pass it explicitly multiple times.